docs/UserGuide.md

```diff
@@ -29,7 +29,17 @@ information quickly with minimal latency.
 ## Quick Start
 
 1. Ensure that you have Java 11 or above installed. 
-2. Download the latest version of `Property Rental Manager` from [here](http://link.to/duke).
+2. Download the latest version of `PropertyRentalManager.jar` from [here](https://github.com/AY2223S1-CS2113-F11-1/tp/releases).
```

Comment:
everything looks good! steps are easy to understand :)

Reply:
I agree. Thanks for also fixing the broken link.